transport_nantes/asso_tn/utils.py

```diff
@@ -96,8 +96,18 @@ def token_valid(encoded_timed_token, test_value_now=None):
     """
     try:
         timed_token = urlsafe_base64_decode(encoded_timed_token).decode()
-    except (TypeError, ValueError):
-        logger.info(f"token_valid: invalid token ({encoded_timed_token})")
+    except (TypeError, ValueError) as e:
+        # On the 31/05/2022 :
+        # While sending a press release, it appears that at least 10 token out
+        # of 24 were invalid. We know this because we got 10 times a
+        # "token_valid: invalid token" error on GET requests to the
+        # 'mailing_list:press_subscription_management' url, seconds after the
+        # send.
+        # In an attempt to reproduce and understand the error, we log more
+        # details about it for the next time it appears
+        # See issue https://github.com/transport-nantes/tn_web/issues/777
```

Comment:
This is good, but note that that issue doesn't have any useful information in it.

Reply:
Added my debug document to issue